### ISSUE TITLE
the SVG canvas size is too big

### ISSUE THREAD

**Issue by toralf** [sysstat/sysstat#286]
A
```
sadf -g -O skipempty,oneday -- -m ALL -n ALL -q -u ALL -w > $svg && firefox $svg
```
gives a a too big SVG canvas size. This is an impediment, especially when a SVG graphics was attached to a GitLab issue tracker is displayed as an inline graphic.
Fun: The SVG does know the correct size, but only as a comment at the end.

**Comment by sysstat** [comment]
Which version of sysstat are you using?

**Comment by toralf** [comment]
The data are generated at the server with 12.2.1, rsynced to my desktop where I do use 12.4.3.
So maybe a small script with an sed statement would make it ?

**Comment by toralf** [comment]
OK, this here works for me:
```
svg=/tmp/graph.svg; sadf -g -O skipempty,oneday -- -m ALL -n ALL -q -u ALL -w > $svg && h=$(tail -n 2 $svg | head -n 1 | cut -f5 -d' ') && sed -i -e "s,height=\"[0-9]*\",height=\"$h\"," $svg &&  firefox $svg
```

**Comment by simonmcnair** [comment]
I am coming across this problem too.  Perhaps just wait until all the information is collated before outputting it to stdout so you do know the size ?
or figure out the number of graphs that are required before you start ?

**Comment by sysstat** [comment]
This issue has now been fixed.
Fix will be part of sysstat version 12.7.6.
